app/extensions/brave/locales/es/preferences.properties

```diff
@@ -24,6 +24,7 @@ autoplay=Autoplay Media
 autoSuggestSites=auto-incluir
 backupLedger=Copia de seguridad de tu cartera
 balanceRecovered=Se ha recuperado un total de {{balance}} y se han transferido a tu cartera Brave.
+banSiteConfirmation=¿Seguro que quieres eliminar este sitio?
```

Comment:
Please remove this line. We are doing translations via Transifex https://github.com/brave/browser-laptop/blob/master/docs/translations.md

Reply:
@NejcZdovc apologies there, i've removed this line.